### DISCUSSION TITLE
Allow moving the URL / Address Bar to the side with vertical tabs

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#803] [Ideas]
### What feature would you like to see?

I would like to have the entire browser UI on the side to save space and for aesthetics, sort of look like Arc browser. There should be an option, perhaps in the settings, to show the URL Bar and browser addons on the sidebar.

### What platform are you seeing the problem on?

macOS

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/538_

**Comment by patrikzudel** [comment]
Yes!

**Comment by Vaengir** [comment]
I would also appreciate the option to display the top bar at the bottom of the window.

**Comment by bengalbroy** [comment]
This!

**Comment by mr-cheffy** [comment]
There's someone making a theme for it

**Reply by YousufSSyed** [reply]
Where can I see it?

**Comment by kekilii** [comment]
That excellent !!! i don't want to open top bar to click some button or check URL

**Comment by 404ishan** [comment]
would love to see this feature being implemented, 100%.
